3.explore-mutations.ipynb

```diff
@@ -152,8 +152,8 @@
        "1           62.0  Female  White        NaN            NaN   \n",
        "\n",
        "                 histological_type     ...     dead  days_survived  \\\n",
-       "0  Untreated primary (de novo) GBM     ...        1          448.0   \n",
-       "1  Untreated primary (de novo) GBM     ...        1           76.0   \n",
+       "0  Untreated primary (de novo) GBM     ...      1.0          448.0   \n",
```

Comment:
Do we know why these changed to floats?

Reply:
Arrgh. It happens if there are any missing values, since pandas is not able to have missing data in int columns.